aws/resource_aws_appmesh_mesh_test.go

```diff
@@ -0,0 +1,119 @@
+package aws
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/appmesh"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAwsAppmeshMesh_importBasic(t *testing.T) {
+	resourceName := "aws_appmesh_mesh.foo"
+	rName := fmt.Sprintf("tf-test-%d", acctest.RandInt())
+
+	resource.Test(t, resource.TestCase{
```

Comment:
Only one service mesh per account allowed right now, so no parallel tests.

Reply:
Tests are still parallelized across function names in our CI environment so a setup similar to `TestAccAWSGuardduty` is required. 👍